<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data correction settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>

      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_data">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="1">
              <layout class="QGridLayout" name="grid_extinction"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_ref_index">
               <property name="text">
                <string>Refractive index correction: OFF</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QGridLayout" name="grid_ref_index"/>
             </item>
             <item row="1" column="3">
              <widget class="QComboBox" name="combobox_ref_index_RE"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_ref_index">
               <property name="text">
                <string>Re{n}</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QComboBox" name="combobox_ref_index_IM"/>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_ref_index">
               <property name="text">
                <string>Im{n}</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_aspect_ratio">
               <property name="text">
                <string>Aspect-ratio correction  [0, 1]: OFF</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_aspectratio_description">
               <property name="text">
                <string>(in the case of non-spherical particles)</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <layout class="QGridLayout" name="grid_aspect_ratio"/>
             </item>
             <item row="3" column="3">
              <widget class="QLineEdit" name="lineEdit_aspect_ratio"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_progbar">
               <property name="text">
                <string>Progress bar:</string>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>110</horstretch>
                 <verstretch>110</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>                                                                                          </string>
               </property>
              </widget>
             </item>

             <item  row="7" column="1">
              <widget class="QPushButton" name="btn_run">
               <property name="text">
                <string>Run</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QPushButton" name="btn_stop">
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
            
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1203</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
